Header header
float64 pan
float64 tilt
float32 timeout_sec
bool initialise
